IssueTracker/IssueTracker.FileSystem/Models/FileModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace IssueTracker.FileSystem.Models;
+public class FileModel
+{
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
Do we need to disable nullable here?

Reply:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/74471664/188217298-a3da9c02-97b1-440c-988b-237d9971d390.png">